README.md

```diff
@@ -314,6 +315,22 @@ past_launches.last.rocket.first_stage.cores.first.land_success # true
 past_launches.last.launch_success # true
 ```
 
+This code snippet shows how to get information on upcoming launches:
```

Comment:
Do you mind following the pattern that the other examples are using? Just to keep it consistent?

Reply:
Sure thing. These should be consistent with the rest of the README now.